src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.FSharp.props

```diff
@@ -0,0 +1,35 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.FSharp.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- If any of these variables are set then we are building under the fsharp.sdk so use their logic -->
+  <PropertyGroup Condition = "'$(_FscTaskAssemblyPath_net)' == '' and '$(_FscTaskAssemblyPath_netcoreapp1_0)' == '' and '$(_FscToolFullPath_net)' == '' and '$(_FscToolFullPath_netcoreapp1_0)' == ''">
+    <NotBuildingUnderFSharpSdk>true</NotBuildingUnderFSharpSdk>
```

Comment:
Actually, I think we should have a looser coupling where there's a way to signal "don't use the FSharp support bundled with the .NET Core Sdk. 
``` 
<PropertyGroup>
  <!-- For compatibility with older versions of FSharp.NET.Sdk that do not set UseBuiltinFSharp. -->
  <UseBundledFSharpSupport Condition="'$(UseBundledFSharpSupport)' and $(_FscTaskAssemblyPath_net)' != ">false</UseBundledFSharpTargets>

  <UseBundedFSharpSupport Condition="'$(UseBundledFSharpSupport)' == ''">true</UseBundledFSharpSupport>
```

So the contract for someone wanting to replace the FSharp toolkit is to set UseBundledFSharpSupport and we only use the check for a known FSharp.NET.Sdk variable to avoid breaking existing versions.